src/transformStateWithClones.js

```diff
@@ -6,8 +6,35 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const array = [];
```

Comment:
```suggestion
  const states = [];
```
don't use obj, arr, etc names for variables
use meaningful ones for them